File: src/main/java/fr/aleclerc/sprint/graph/plush/PlushState.java
package fr.aleclerc.sprint.graph.plush;

public class PlushState {
	private Plush plush;
	private User owner;
	
	public Plush getPlush() {
		return plush;
	}
	public void setPlush(Plush plush) {
		this.plush = plush;
	}
	public User getOwner() {
		return owner;
	}
	public void setOwner(User owner) {
		this.owner = owner;
	}
	@Override
	public String toString() {
		return "PlushState [plush=" + plush + ", owner=" + owner + "]";
	}
	
	
	
	
	
}

File: frontend/src/main/frontend/src/app/models/plush-state.ts
import { User } from './user';
import { Plush } from './plush';
export class PlushState {

    public static create(object: any): PlushState {
        if (object == null) {
            return null;
        }
        return new PlushState(object.plush, object.owner);
    }

    public static createFromArray(object: any): Array<PlushState> {
        if (object != null && object instanceof Array) {
            return object.map(v => new PlushState(v.plush, v.owner));
        }
        return null;
    }


    constructor(public plush: Plush, public owner: User) {
    }



}